src/masternodes/operators.h

```diff
@@ -30,7 +30,16 @@ struct CCreateOperatorMessage {
     CScript operatorAddress;
     std::string operatorName;
     std::string operatorURL;
-    // OperatorState operatorState;
+    uint8_t operatorState{0};
+
+    CCreateOperatorMessage(CScript operatorAddress_, std::string operatorName_, std::string operatorURL_, OperatorState operatorState_)
+                            :  operatorAddress(operatorAddress_)
+                            ,  operatorName(operatorName_)
+                            ,  operatorURL(operatorURL_)
```

Comment:
Use `std::move` for these

Reply:
was using `std::forward` for this. But seems it is not perfect for this situation(when lval is passed).
will change to `std::move`with the next commit, better.